opal/mca/pmix/pmix3x/pmix/src/mca/pdl/configure.m4

```diff
@@ -25,15 +27,15 @@ AC_DEFUN([MCA_pmix_pdl_CONFIG],[
     # (we still need to configure them all so that things like "make
     # dist" work", but we just want the MCA system to (artificially)
     # conclude that it can't build any of the components.
-    AS_IF([test "$enable_dlopen" = "no"],
+    AS_IF([test "$PMIX_ENABLE_DLOPEN_SUPPORT" = "0"],
```

Comment:
If you're going to change the variable to be numeric, you might as well change `=` to `-eq`.